### DISCUSSION TITLE
The next-server occupied 4gb memory and the system becomes very slow in a local development environment in a Next.js application?

### DISCUSSION THREAD

**Discussion by raguct25** [vercel/next.js#68145] [Help]
### Summary

I have developed a Next.js application with multiple packages. When I run npm run dev, the system becomes very slow, especially when navigating between components or trigger any actions.

I have run next.js new application its occupied just **400 mb** only

![image](https://github.com/user-attachments/assets/43018d2c-aca9-4e37-afb0-678178311f87)

When I run my nextjs application it has occupied **4 GB**

![image](https://github.com/user-attachments/assets/5709a90d-262d-4dd2-9a2f-6c5f93cac768)

Used Package Details:

Dependencies Details:
![dependency-package](https://github.com/user-attachments/assets/cfef41e7-46aa-45d2-ad37-b1573a34e197)

Dev Dependencies Details:
![dev-dependency-packages](https://github.com/user-attachments/assets/da8f42b6-76c8-4483-b82d-13b43df46d93)

Is it possible to fix this issue with any specific implementation? How can I improve the local development speed?

Are there any standards or best practices I should follow for using packages in my Next.js application to avoid performance issues?

**Comment by UlugbekAsadov** [comment]
@raguct25 Hi. Did you find any solution?
